build.sh

```diff
@@ -1,7 +1,50 @@
 #!/bin/bash
 
-NGINX_VERSION=${NGINX_VERSION:-1.25.1}
+#########################################################################
+# Builds a particular NGINX version for a particular Linux OS and version
+#########################################################################
+
+NGINX_VERSION=${NGINX_VERSION:-1.25.5}
 NGINX_TARBALL=nginx-${NGINX_VERSION}.tar.gz
+LINUX_DISTRO=${LINUX_DISTRO:-alpine}
+
+if [ "$LINUX_DISTRO" != 'alpine' ] &&
+   [ "$LINUX_DISTRO" != 'debian11' ] &&
+   [ "$LINUX_DISTRO" != 'debian12' ] &&
+   [ "$LINUX_DISTRO" != 'ubuntu20' ] &&
+   [ "$LINUX_DISTRO" != 'ubuntu22' ] &&
+   [ "$LINUX_DISTRO" != 'ubuntu24' ] &&
+   [ "$LINUX_DISTRO" != 'amazon2' ] &&
+   [ "$LINUX_DISTRO" != 'amazon2023' ] &&
+   [ "$LINUX_DISTRO" != 'centos7' ] &&
+   [ "$LINUX_DISTRO" != 'centosstream9' ]; then
+  echo "$LINUX_DISTRO is not a supported Linux distribution"
+  exit 1
+fi
+
+function getLibraryPrefix() {
```

Comment:
I also used the same file prefixes as previously, so manage that using a function.